modules/checkout/src/Plugin/Commerce/CheckoutFlow/MultistepDefault.php

```diff
@@ -20,13 +20,16 @@ public function getSteps() {
       'login' => [
         'label' => $this->t('Login'),
         'previous_label' => $this->t('Return to login'),
+        'has_cart_summary' => FALSE,
       ],
       'order_information' => [
         'label' => $this->t('Order information'),
+        'has_cart_summary' => TRUE,
       ],
       'review' => [
         'label' => $this->t('Review'),
         'next_label' => $this->t('Continue to review'),
+        'has_cart_summary' => FALSE,
```

Comment:
let's have it TRUE for review for now. 
We'd make it FALSE if we provide a different summary for the review page alone.